client/packages/system/src/Vaccination/Components/VaccineCardTable.tsx

```diff
@@ -165,7 +165,7 @@ const VaccinationCardComponent = ({
           // Only show age label for first of each "block", when repeated
           return sameAsPrev ? null : getAgeLabel(rowData);
         },
-        width: 140,
+        width: '5%',
```

Comment:
Hm, so looks like `5` of the 5% isn't actually for a specific purpose, just that its smaller than actual width so stretching to min-content... in this case, I think safer to go with 1% (as its more obviously not the actual width) and add a comment:

```suggestion
        // Hack for "min-content"
        width: '1%',
```

I have added this to table refactor issue, something else to consider in new table API definition: https://github.com/msupply-foundation/open-msupply/issues/4744

Reply:
Resolved here 6cdeb7c49460d0136127b63faa407c204e7fb20a